app/main/index.js

```diff
@@ -36,15 +36,23 @@ let win = null;
 // create a user config
 const config = new Config();
 
+const closeWindow = () => {
```

Comment:
We should consider modifying the eslint `no-use-before-define` rule to allow hoisting